app/views/dogs/index.html.erb

```diff
@@ -41,12 +41,12 @@
       <p>Size: <%= dog.size %></p>
       <p>Neutered: <%= dog.neutered ? 'true' : 'false' %></p>
       <!-- Add photo display logic here if we decide to -->
-      <%= button_to "Edit #{dog.name}", edit_dog_path(@user.user_id, dog.dog_id) %>
+      <%= button_to "Edit #{dog.name}", edit_user_dog_path(@user.user_id, dog.dog_id) %>
```

Comment:
Great update with the route here since `Dogs` are nested under `Users` in our routing.